server/agones.cpp

```diff
@@ -0,0 +1,111 @@
+#include "agones.hpp"
+
+namespace dfAgones = agones;
+
+#include <agones/sdk.h>
+#include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
+
+static std::shared_ptr<agones::SDK> sdk;
+
+// send health check pings
+static void DoHealth()
+{
+	while (true)
+	{
+		bool ok = sdk->Health();
+		if (!ok) {
+			std::cout << "health ping failed\n";
+		}
+		std::this_thread::sleep_for(std::chrono::seconds(2));
+	}
+}
+
+// watch GameServer Updates
+static void WatchUpdates()
+{
+	std::cout << "Starting to watch GameServer updates...\n"
+			  << std::flush;
+	sdk->WatchGameServer([](const agones::dev::sdk::GameServer &gameserver) {
+		std::cout << "GameServer Update:\n"								   //
```

Comment:
Nit: There is an empty comment in this line, prepended by lots of whitespace characters